test/opencensus_phoenix_test.exs

```diff
@@ -2,6 +2,5 @@ defmodule OpencensusPhoenixTest do
   use ExUnit.Case
 
   test "greets the world" do
-
```

Comment:
Tests would be probably useful ;)

Reply:
Yea, I'm guessing it is easier to test then I'm making it out in my head (needing a phoenix app to instrument for the testing). I'll poke around on that soon.

Reply:
You do not need Phoenix app for that TBH. You can test it like any other plug TBH.